contracts/orbital-core/src/contract.rs

```diff
@@ -1,119 +1,171 @@
+use crate::{
+    admin_logic::admin,
+    state::{OrbitalDomainConfig, UserConfig, USER_NONCE},
+    user_logic::user,
+    utils::{extract_ica_identifier_from_port, get_ica_identifier, OpenAckVersion},
+};
 #[cfg(not(feature = "library"))]
 use cosmwasm_std::entry_point;
 use cw2::set_contract_version;
-use cw_ownable::{
-    assert_owner, get_ownership, initialize_owner, update_ownership, Action, OwnershipError,
+use cw_ownable::{get_ownership, initialize_owner};
+use neutron_sdk::{
+    bindings::{msg::NeutronMsg, query::NeutronQuery},
+    sudo::msg::SudoMsg,
+    NeutronResult,
 };
 
 use crate::{
-    account_types::UncheckedOrbitalDomainConfig,
-    error::ContractError,
-    msg::{ExecuteMsg, InstantiateMsg, QueryMsg},
-    state::{UserConfig, ORBITAL_DOMAINS, USER_CONFIGS},
+    msg::{ExecuteMsg, InstantiateMsg, MigrateMsg, QueryMsg},
+    state::{CLEARING_ACCOUNTS, ORBITAL_DOMAINS, USER_CONFIGS},
 };
 use cosmwasm_std::{
-    to_json_binary, Addr, Binary, BlockInfo, Deps, DepsMut, Env, MessageInfo, Response, StdResult,
+    to_json_binary, Addr, Binary, Deps, DepsMut, Env, MessageInfo, Reply, Response, StdError,
+    StdResult, Uint64,
 };
 
 pub const CONTRACT_NAME: &str = "orbital-core";
 pub const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
 
+pub type OrbitalResult = NeutronResult<Response<NeutronMsg>>;
```

Comment:
nit: I would use it either everywhere or nowhere. For example for instantiate you are using `OrbitalResult`, but then for execute you are using `NeutronResult<Response<NeutronMsg>>`.

Or tbh, just use `Result<Response<NeutronMsg>, ContractError>` everywhere. Just to keep it consistent

Reply:
yeah that's something I forgot about! cleaning up

Reply:
removed OrbitalResult entirely